@navikt/core/css/accordion.css

```
@@ -5,70 +5,142 @@
-  background: var(--ac-accordion-header-bg-hover, var(--a-surface-transparent));
+  background: var(--ac-accordion-header-bg-hover, var(--a-grayalpha-200));
+  text-decoration: underline;
+}
+
+.navds-accordion__header:hover > .navds-accordion__header-chevron {
+  background-color: var(--ac-accordion-header-bg-hover, var(--a-surface-hover));
+  border-radius: 4px;
+  height: 24px;
+  align-self: center;
+}
+
+.navds-accordion__header > .navds-accordion__icon-wrapper {
+  display: grid;
+  place-content: center;
+  cursor: pointer;
+  margin: 0;
+  border: none;
+  border-radius: var(--ac-expansioncard-button-border-radius, var(--a-border-radius-medium));
+  height: var(--a-spacing-6);
+  width: var(--a-spacing-6);
+  font-size: 1.5rem;
+  align-self: center;
+}
+
+.navds-accordion__header:hover > .navds-accordion__icon-wrapper {
+  background-color: var(--ac-accordion-header-bg-hover, var(--a-surface-hover));
+}
+
+:where(.navds-accordion__header):hover {
+  background-color: var(--ac-accordion-header-bg-hover, var(--a-surface-hover));
 }
 
 .navds-accordion__header-content {
   overflow: hidden;
   text-overflow: ellipsis;
 }
 
-.navds-accordion__item--open > .navds-accordion__header {
-  background-color: var(--ac-accordion-header-bg-open, var(--ac-accordion-item-bg-open, var(--a-surface-action-subtle)));
-  border-color: var(--ac-accordion-item-border-open, var(--a-surface-action-subtle));
+.navds-accordion__item--open.navds-accordion__item--neutral {
+  background-color: var(--ac-accordion-header-bg-open, var(--ac-accordion-item-bg-open, var(--a-grayalpha-200)));
 }
 
-.navds-accordion__item--open > :where(.navds-accordion__header:hover) {
-  background-color: var(--ac-accordion-header-bg-open-hover, var(--a-surface-action-subtle));
+.navds-accordion__item--open .navds-accordion__header {
+  border: none;
 }
 
 .navds-accordion__content {
-  padding: var(--a-spacing-3) var(--a-spacing-3) 18px;
+  padding: 0 var(--a-spacing-3) var(--a-spacing-5) var(--a-spacing-11);
   border-bottom: 2px solid var(--ac-accordion-content-border, var(--a-border-divider));
+  animation: fadeAccordionContent 250ms ease;
 }
 
 .navds-accordion__item--open > .navds-accordion__header:hover + * .navds-accordion__content {
   border-color: var(--ac-accordion-content-border-open, var(--a-border-default));
 }
 
-.navds-accordion__expand-icon {
+.navds-accordion__header-chevron {
   font-size: 1.5rem;
   height: 1.75rem;
   flex-shrink: 0;
+  transition: transform 150ms ease-in-out;
 }
 
-.navds-accordion__item--open > .navds-accordion__header > .navds-accordion__expand-icon {
-  transform: rotateZ(180deg);
+:where(.navds-accordion__header):hover :where(.navds-accordion__header-chevron) {
+  transform: translateY(1px);
 }
 
-.navds-accordion__expand-icon--filled {
-  display: none;
+.navds-accordion__item--open :where(.navds-accordion__header-chevron) {
+  transform: translateY(0) rotate(180deg);
```

Rotere pil på "insiden"
```suggestion
  transform: translateY(0) rotate(-180deg);
```

---

Confused here, jeg får rar "360" spin on hover om jeg lager denne endringen

---

aaah nvm, henger sammen med under... got it... ja, retningen den spinner